### PR TITLE
[OPS-7389] allow later versions of test packages

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Another crack at https://github.com/UN-OCHA/taas/pull/61

I also changed the python version number in the Makefile to match the current latest base image `FROM unocha/alpine-base-s6-python3:3.8`

### CODE REVIEW

**Comment by lazysoundsystem** [comment]
@teodorescuserban - not sure why upgrading python version to 3.8 in the `Makefile` and `.travis.yml` caused the earlier tests to fail - do you know where python3.6 is specified? https://travis-ci.com/github/UN-OCHA/taas/builds/222403540

All the packages whose versions I changed are concerned with test coverage only. So no functionality should be affected.

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,8 @@ GIT=git
 SED=sed
 
 venv: requirements.txt
+  # 3.6 appears to be required to pass the travis tests.
+  # 3.8 is the version used in the base docker image.
 	virtualenv --python=python3.6 --system-site-packages venv
```
Comment (teodorescuserban) [inline Makefile:19]: Here it is specified, @lazysoundsystem

Comment (teodorescuserban) [reply Makefile:19]: and in `.travis.yml` line 5

**Review by teodorescuserban — APPROVED** [review]
looks good to me. tested on jenkins.

**Comment by lazysoundsystem** [comment]
Thanks for merging @teodorescuserban and glad it's working. 

Still curious, though: I adjusted the python version to 3.8 in https://github.com/UN-OCHA/taas/pull/62/commits/22108410100443a5ad0525591353a0106b85f23c (the Makefile) and in https://github.com/UN-OCHA/taas/pull/62/commits/58e44792fdfc35d95854d289f3cfc39ef2a72eb2 (travis.yml) but with those the tests failed:
https://travis-ci.com/github/UN-OCHA/taas/builds/222404945
```
$ source ~/virtualenv/python3.6/bin/activate

$ python --version

Python 3.6.7

$ pip --version

pip 20.1.1 from /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/pip (python 3.6)

0.07s$ make

virtualenv --python=python3.8 --system-site-packages venv

The path python3.8 (from --python=python3.8) does not exist

Makefile:17: recipe for target 'venv' failed

make: *** [venv] Error 3

The command "make" failed and exited with 2 during .

Your build has been stopped.
```

As it works with 3.6, it's maybe not important (and definitely low priority), but I was wondering why the python version in the travis tests (above) is:
```
$ python --version

Python 3.6.7
```
when the .travis.yml file specifies 3.8.

**Comment by teodorescuserban** [comment]
@lazysoundsystem, I don't see `.travis.yml` modified. This means that, according to [line 5](https://github.com/UN-OCHA/taas/blob/master/.travis.yml#L5), travis is using a baseimage that includes python 3.6, not 3.8. Hence the virtualenv failure to link in the new virtual envirnment python 3.8.

**Comment by lazysoundsystem** [comment]
That was my question - https://github.com/UN-OCHA/taas/pull/62/commits/58e44792fdfc35d95854d289f3cfc39ef2a72eb2 was meant to bump travis to 3.8, but the build failed.

Screenshot from 2021-04-12 12-47-58

**Comment by teodorescuserban** [comment]
Unfortunately I dont seem to be able to see the actual travis build for that particular commit.
It might have been that only .travis.yml python version was bumped but not the env command itself.

We should try again changing both of them in the same commit and see what happens. It is unlikely that travis wont be able to provide a python 3.8 base image.

Serban Teodorescu

**Comment by lazysoundsystem** [comment]
I think the build was https://travis-ci.com/github/UN-OCHA/taas/builds/222404945 - I got the link from clicking on the red cross showing the test failed.

Anyway, seems like that was a mistake as https://github.com/UN-OCHA/taas/pull/64 passed the tests!

**Pull request merged**